bundles/org.openhab.binding.plugwiseha/src/main/resources/OH-INF/thing/channels.xml

```diff
@@ -223,6 +263,10 @@
 		<label>Domestic Hot Water Setpoint Temperature</label>
 		<description>Gets the temperature of the domestic hot water setpoint</description>
 		<category>heating</category>
+		<tags>
+			<tag>Measurement</tag>
```

Comment:
```suggestion
			<tag>Setpoint</tag>
```

Reply:
And given your suggestion here, I would assume that is what you meant for the one above. Or??